src/main.rs

```diff
@@ -104,6 +104,13 @@ fn notmain() -> Result<i32, anyhow::Error> {
         );
     }
 
+    elf.find_section_by_name(".text").ok_or_else(|| {
+        anyhow!(
+            "`.text` section is missing, please make sure that the linker script \
+            was passed to the linker"
```

Comment:
maybe also suggest looking into `.cargo/config.toml`? setting `RUSTFLAGS` also breaks linking like this